### PR TITLE
Fix UTF-8 filepath for RRD

### PR DESCRIPTION
**Pull request by deniszh**
Fixing https://github.com/graphite-project/docker-graphite-statsd/issues/63

### CODE REVIEW

**Comment by deniszh** [comment]
@piotr1212, could you please check this? I'm not sure that it's a proper fix...

File: webapp/graphite/readers/rrd.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class RRDReader(BaseReader):
     def _convert_fs_path(fs_path):
         if isinstance(fs_path, six.text_type):
             fs_path = fs_path.encode(sys.getfilesystemencoding())
-        return os.path.realpath(fs_path)
+        return os.path.realpath(fs_path).decode("utf-8")
```
[the remark this answers is not in the document]
Comment (DanCech) [reply webapp/graphite/readers/rrd.py]: It should, but as far as I can tell it doesn't

**Comment by deniszh** [comment]
Ah, test is there but it tests pretty much nothing, d-oh - https://github.com/graphite-project/graphite-web/blob/master/webapp/tests/test_readers_rrd.py
Will fix that.

**Comment by deniszh** [comment]
Thanks! I fixed a patch, will try to create RRD file and create a proper test.

**Comment by piotr1212** [comment]
So this patch breaks Python 2 compatibility because .decode() returns an 'unicode' object while rrdtool is expecting an Py 2 'str'.

Also note that the return type of os.path.realpath depends on the parameter, so if skip the encode we don't need to decode.

This should work on all Python versions https://github.com/piotr1212/graphite-web/commit/bff2a27ff84d992160bb036c1c4e63af6d4d2ccd

**Comment by deniszh** [comment]
Well, I still have a hope to create proper RRD test.

**Comment by piotr1212** [comment]
I'll try to make some tests this evening.

**Comment by piotr1212** [comment]
Well, I tested it manually and it worked in Py2 and Py3, so I think it is OK to merge. The tests are more broken than I expected, so I might not be ready today.

**Comment by deniszh** [comment]
Closed in the favor of https://github.com/graphite-project/graphite-web/pull/2383

**Pull request closed without merging**